requirements/runtime.txt

```diff
@@ -1,3 +1,4 @@
+fvcore
```

Comment:
Please move it to the `optional` requirements file, since we use it only in the flops analysis tool.